### ISSUE TITLE
Access to component context through Output ?

### ISSUE THREAD

**Issue by raphael22** [VadimDez/ng2-pdf-viewer#40]
**Issue** : Could not access to my component variables on after-load callback function.
```javascript
// my-component.ts
onLoaded(pdf) {
  this.loading = false // <- loading unknown
}
```

**Proposal**: Use Output instead of Input with pdf as return value.

```javascript
// pdf-viewer.component.ts
@Output() afterLoadCompleteEmitter = new EventEmitter()
...
if (this.afterLoadComplete && typeof this.afterLoadComplete === 'function') {
  this.afterLoadComplete(pdf);
}
...
afterLoadComplete(pdf) {
  this.afterLoadCompleteEmitter.emit({ value: pdf })
}


// my.component.html
<pdf-viewer [src]="selected.value"
            (afterLoadComplete)="onLoaded($event)"
            [show-all]="true"
            [page]="1" 
            [original-size]="true">
</pdf-viewer>

// my-component.ts
onLoaded(event) {
  const pdf = event.value
  this.loading = false // <- loading known
}
```

**Comment by raphael22** [comment]
I've just see your binding in src example.
```javascript
    this.afterLoadComplete = this.afterLoadComplete.bind(this);
```
But Output wouldn't be a better workaround ?

**Comment by VadimDez** [comment]
Resolved by #47
